src/Orc.NuGetExplorer.Tests/Orc.NuGetExplorer.Tests.csproj

```diff
@@ -9,11 +9,12 @@
     <Description></Description>
     <PackageTags></PackageTags>
     <SonarQubeExclude>true</SonarQubeExclude>
+    <Nullable>annotations</Nullable>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="ApprovalTests" Version="5.7.3" />
-    <PackageReference Include="Catel.Core" Version="5.12.22" />
+    <PackageReference Include="Catel.Core" Version="6.0.0-alpha1014" />
```

Comment:
remove reference to Catel.Core, it's automatically included via ORc.NuGetExplorer